### ISSUE TITLE
Provide typings

### ISSUE THREAD

**Issue by paales** [martpie/next-transpile-modules#190]
**Is your feature request related to a problem? Please describe.**
Provide typings

**Describe the solution you'd like**
```ts
declare module 'next-transpile-modules' {
  import type { NextConfig } from 'next/dist/next-server/server/config'

  export default function (
    modules: string[],
    options?: {
      resolveSymlinks?: boolean
      debug?: boolean
      __unstable_matcher?: (path: string) => boolean
    },
  ): (nextConfig: NextConfig) => NextConfig
}
```

**Comment by martpie** [comment]
TSDoc is already provided, what am I missing?

https://github.com/martpie/next-transpile-modules/blob/e7222640dd72c0b4702ba5abbe79e261182a454e/src/next-transpile-modules.js#L75-L79

**Comment by martpie** [comment]
I'm asking because I'd say the only missing thing is if you use `next.config.ts`, but I don't know anyone who does that.

**Comment by paales** [comment]
For https://github.com/martpie/next-transpile-modules/issues/189 I was wrapping `next-transpile-modules` and was building it in typescript and having type definitions here was nice :)

**Comment by martpie** [comment]
I will see if there is a way to generate a `.d.ts` based on the TSDoc then

**Comment by martpie** [comment]
Fixed in 2a5d7bb.

May not be extremely accurate, so any JSDoc update is appreciated.

**Comment by paales** [comment]
🎉
